postgres/problems/problem_12.md

```
@@ -1,23 +1,21 @@
-  まず注文 ID (`order_id`) 昇順、その中で割合が **大きい順（降順）** に並べること
+* WHERE 内でサブクエリを使用すること
+* GROUP BY、HAVING、JOIN などの追加句は使用しないこと
 
 ## 想定出力例（先頭 5 行）
-
-| order_id | product_id | quantity_ratio_percent |
-|----------|------------|------------------------|
-|      100 |        101 |                  60.00 |
-|      100 |        234 |                  40.00 |
-| …        |        …   |                   …    |
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)